Alpine Package Build Tests Failing During Release Production

The alpine package test builds are failing with the below..

```
> Task :jdk:alpine:compilePackageTestJava
Note: /home/jenkins/workspace/build-scripts/release/create_installer_linux/jdk/alpine/src/packageTest/java/packaging/AlpineFlavours.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :jdk:alpine:processPackageTestResources
> Task :jdk:alpine:packageTestClasses

> Task :jdk:alpine:checkJdkAlpine

ApkOperationsTest > packageSuccessfullyInstalled(String, String) > "alpine":"3.17" FAILED
    java.lang.IllegalStateException at ApkOperationsTest.java:50

ApkOperationsTest > packageSuccessfullyInstalled(String, String) > "alpine":"3.19" FAILED
    java.lang.IllegalStateException at ApkOperationsTest.java:50

ApkOperationsTest > packageSuccessfullyInstalled(String, String) > "alpine":"3.18" FAILED
    java.lang.IllegalStateException at ApkOperationsTest.java:50

ApkOperationsTest > packageSuccessfullyInstalled(String, String) > "alpine":"latest" FAILED
    java.lang.IllegalStateException at ApkOperationsTest.java:50

ApkOperationsTest > packageSuccessfullyInstalled(String, String) > "alpine":"edge" FAILED
    java.lang.IllegalStateException at ApkOperationsTest.java:50

5 tests completed, 5 failed

> Task :jdk:alpine:checkJdkAlpine FAILED

[Incubating] Problems report is available at: file:///home/jenkins/workspace/build-scripts/release/create_installer_linux/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.
```

--- a/linux_new/jdk/alpine/build.gradle
+++ b/linux_new/jdk/alpine/build.gradle
@@ -6,7 +6,7 @@ import org.gradle.process.ExecOperations
 
 ext {
 	junitVersion = "6.0.3"
-	testcontainersVersion = "1.20.3"
+	testcontainersVersion = "1.21.4"
 	assertjCoreVersion = "3.26.3"
 }
 

--- a/linux_new/jdk/alpine/src/packageTest/java/packaging/AlpineFlavours.java
+++ b/linux_new/jdk/alpine/src/packageTest/java/packaging/AlpineFlavours.java
@@ -32,7 +32,7 @@ public class AlpineFlavours implements ArgumentsProvider {
 		 * Alpine policy: current (alive) releases and development version.
 		 *     (https://alpinelinux.org/releases/)
 		 */
-		
+
 		String containerRegistry = "";
 
 		if (System.getenv("containerRegistry") == null) {
@@ -45,9 +45,9 @@ public class AlpineFlavours implements ArgumentsProvider {
 		return Stream.of(
 			Arguments.of(containerRegistry + "alpine", "edge"),
 			Arguments.of(containerRegistry + "alpine", "latest"),
-			Arguments.of(containerRegistry + "alpine", "3.19"),
-			Arguments.of(containerRegistry + "alpine", "3.18"),
-			Arguments.of(containerRegistry + "alpine", "3.17")
+			Arguments.of(containerRegistry + "alpine", "3.21"),
+			Arguments.of(containerRegistry + "alpine", "3.22"),
+			Arguments.of(containerRegistry + "alpine", "3.23")
 		);
 	}
 }

--- a/linux_new/jre/alpine/build.gradle
+++ b/linux_new/jre/alpine/build.gradle
@@ -6,7 +6,7 @@ import org.gradle.process.ExecOperations
 
 ext {
 	junitVersion = "6.0.3"
-	testcontainersVersion = "1.20.3"
+	testcontainersVersion = "1.21.4"
 	assertjCoreVersion = "3.26.3"
 }
 

